onnxruntime/contrib_ops/rocm/bert/layer_norm.cuh

```diff
@@ -109,6 +109,61 @@ __device__ inline void LayerNorm(
   }
 }
 
+template <typename U, typename V, int TPB>
+__device__ inline void SimplifiedLayerNorm(
+    const U& thread_data, const int ld, const int offset, const V* gamma, const U epsilon, V* output) {
```

Comment:
What is `ld`?

Reply:
It's `hidden_size`, not sure why we named it `ld,` I just keep using the variable name in cuda ep.